leshan-integration-tests/src/test/java/org/eclipse/leshan/integration/tests/util/QueueModeLeshanServer.java

```diff
@@ -47,29 +47,30 @@
     private final LwM2mRequestSender lwM2mRequestSender;
     private final MessageStore messageStore;
 
-    public QueueModeLeshanServer(CoapServer coapServer, ClientRegistry clientRegistry,
-            ObservationRegistry observationRegistry, SecurityRegistry securityRegistry,
-            LwM2mModelProvider modelProvider, LwM2mRequestSender lwM2mRequestSender,
-            MessageStore inMemoryMessageStore) {
-
-        this.coapServer = coapServer;
-        this.clientRegistry = clientRegistry;
-        this.observationRegistry = observationRegistry;
-        this.securityRegistry = securityRegistry;
-        this.modelProvider = modelProvider;
-        this.lwM2mRequestSender = lwM2mRequestSender;
-        this.messageStore = inMemoryMessageStore;
+    public QueueModeLeshanServer(CoapServer tCoapServer, ClientRegistry tClientRegistry,
+            ObservationRegistry tObservationRegistry, SecurityRegistry tSecurityRegistry,
+            LwM2mModelProvider tModelProvider, LwM2mRequestSender tLwM2mRequestSender,
+            MessageStore tInMemoryMessageStore) {
+
+        this.coapServer = tCoapServer;
+        this.clientRegistry = tClientRegistry;
+        this.observationRegistry = tObservationRegistry;
+        this.securityRegistry = tSecurityRegistry;
+        this.modelProvider = tModelProvider;
+        this.lwM2mRequestSender = tLwM2mRequestSender;
+        this.messageStore = tInMemoryMessageStore;
```

Comment:
Personnaly, I prefer QueueModeLeshanServer.this.observationRegistry.
(the "t" stand for what ?)